event_operating_unit/tests/test_operating_unit_user_id.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2018 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo.tests.common import SavepointCase
+
+
+class TestOperatiingUnitUserId(SavepointCase):
+
+    def setUp(self):
+        super().setUp()
+
+        self.ou1 = self.env.ref('operating_unit.b2c_operating_unit')
+        self.ou2 = self.env.ref('operating_unit.b2b_operating_unit')
+
+        self.user1 = self.env['res.users'].create({
+            'name': 'User 1',
+            'login': 'user1',
+            'operating_unit_ids': [(4, self.ou1.id)]
+        })
+
+        self.user2 = self.env['res.users'].create({
+            'name': 'User 2',
+            'login': 'user2',
+            'operating_unit_ids': [(4, self.ou2.id)]
+        })
+
+        self.user3 = self.env['res.users'].create({
+            'name': 'User 3',
+            'login': 'user3',
+            'operating_unit_ids': [(4, self.ou1.id), (4, self.ou2.id)]
+        })
+
+        self.event1 = self.env['event.event'].create({
+            'name': 'Testevent',
+            'operating_unit_id': self.ou1.id,
+            'date_begin': '2018-03-01 10:00:00',
+            'date_end': '2018-03-01 11:00:00'
+        })
+
+        self.event2 = self.env['event.event'].create({
+            'name': 'Testevent',
+            'operating_unit_id': self.ou2.id,
+            'date_begin': '2018-03-01 10:00:00',
+            'date_end': '2018-03-01 11:00:00'
+        })
+
+    def test_compute(self):
+        res = self.event1.operating_unit_user_ids
```

Comment:
res is a `res.users` recordset, please call the var users, or something with more sense.